docs/docs/general/02-Getting Started/write-your-first-instruction.md

```diff
@@ -2,7 +2,7 @@
 sidebar_position: 3
 ---
 
-# Write Your First Automation
+# Write Your First Instruction
```

Comment:
Why do you think this is better than `Automation`?

I saw that we still talk about `automation` on this page. So if we change it here, I believe we should make it consistent across the page.

Reply:
Currently we are trying to get our vocabulary sorted out.

The current reasoning is this:
-> Instruction is one singular action that is performed: For example a mouse click
-> Automation: A set of instructions that automate something: For example 2 FA with Github and Android Phone

You are right that we talk of automation on the whole page. We have a set of tickets in JIRA to straighten this out. We just have to start somewhere 😉

Reply:
Understood :) 

But to me it still looks like this should stay `Automation` as it is not a single action that is written here...